### ISSUE TITLE
When playing a podcast and then minimizing spmc audio slows down

### ISSUE THREAD

**Issue by donaldsmith2060** [koying/SPMC#117]
I'm not sure if this is any audio or just when I play a podcast from the iTunes podcast plugin, but when I play audio and then goto the home screen audio slows down and gets choppy.

**Comment by ghost** [comment]
Debug log?

**Comment by donaldsmith2060** [comment]
Where is it located, and how do I get it off the shield?

**Comment by donaldsmith2060** [comment]
[spmclog.zip](https://github.com/koying/SPMC/files/180987/spmclog.zip)

**Comment by koying** [comment]
Cannot reproduce.
Any special in your setup?

**Comment by donaldsmith2060** [comment]
Android 6, shield

**Comment by typeiionair** [comment]
yep I have the same issue. mine is different. I don't think it matters what audio is playing. like donald said..... Where is it located, and how do I get it off the shield?
